src/main/common/streambuf.c

```diff
@@ -20,6 +20,13 @@
 
 #include "streambuf.h"
 
+sbuf_t *sbufInit(sbuf_t *sbuf, uint8_t *ptr, uint8_t *end)
```

Comment:
in most cases `uint8_t *ptr, int len` would be easier to use. The `end` is internal optimization of `sbuf`

Reply:
I'll need to check. I thought this was probably more common, but not sure.